gcp-cups-connector/gcp-cups-connector.go

```diff
@@ -42,17 +42,19 @@ func main() {
 			Usage: "Log to STDERR, in addition to configured logging",
 		},
 	}
-	app.Action = func(context *cli.Context) {
-		os.Exit(connector(context))
+	app.Action = func(context *cli.Context) error {
+                return_code, _ := connector(context)
+		os.Exit(return_code)
```

Comment:
Is there a way to wrap the return code so that cli will understand it?
https://github.com/urfave/cli/blob/master/CHANGELOG.md#deprecated-cli-app-runandexitonerror suggests using a `cli.ExitCoder`.